Bio/SearchIO/BlastIO/blast_xml.py

```diff
@@ -665,9 +693,11 @@ def __init__(self, out, encoding="utf-8", indent=" ", increment=2):
 
     def startDocument(self):
         """Start the XML document."""
-        self.write(u'<?xml version="1.0"?>\n'
-                   '<!DOCTYPE BlastOutput PUBLIC "-//NCBI//NCBI BlastOutput/EN" '
-                   '"http://www.ncbi.nlm.nih.gov/dtd/NCBI_BlastOutput.dtd">\n')
+        self.write(
+            u'<?xml version="1.0"?>\n'
+            '<!DOCTYPE BlastOutput PUBLIC "-//NCBI//NCBI BlastOutput/EN" '
+            '"http://www.ncbi.nlm.nih.gov/dtd/NCBI_BlastOutput.dtd">\n'
+        )
```

Comment:
This is odd - @bow why would you have set the first bit of the string explicitly as unicode but not the rest?

This won't matter soon as we're planning only one final release with Python 2 support, after which we can drop all the unicode prefixes.